packages/genesys-spark-components/src/components/stable/gux-tooltip/gux-tooltip.scss

```diff
@@ -31,7 +35,10 @@
   font-size: var(--gse-ui-tooltip-text-fontSize);
   font-weight: var(--gse-ui-tooltip-text-fontWeight);
   line-height: var(--gse-ui-tooltip-text-lineHeight);
-  color: var(--gse-ui-tooltip-light-foregroundColor);
+  color: var(
```

Comment:
![image](https://github.com/user-attachments/assets/bac2d633-1798-42f2-a1d0-cdfc37495292)

I would expect this text to be black in the tooltip